pkg/ingester/ingester.go

```diff
@@ -380,15 +382,20 @@ func newIngester(cfg Config, limits *validation.Overrides, registerer prometheus
 }
 
 // New returns an Ingester that uses Mimir block storage.
-func New(cfg Config, limits *validation.Overrides, ingestersRing ring.ReadRing, partitionRingWatcher *ring.PartitionRingWatcher, activeGroupsCleanupService *util.ActiveGroupsCleanupService, registerer prometheus.Registerer, logger log.Logger) (*Ingester, error) {
+func New(
+	cfg Config, limits *validation.Overrides,
+	ingestersRing ring.ReadRing, partitionRingWatcher *ring.PartitionRingWatcher,
+	activeGroupsCleanupService *util.ActiveGroupsCleanupService,
+	costAttributionCleanupService *util.CostAttributionCleanupService,
+	registerer prometheus.Registerer, logger log.Logger) (*Ingester, error) {
```

Comment:
Same here, can you drop these to separate lines? Otherwise it just looks like a cloud of params.

Reply:
done